tests/python/gaia-ui-tests/gaiatest/tests/functional/clock/test_clock_run_stopwatch_laps.py

```diff
@@ -0,0 +1,54 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.clock.app import Clock
+
+class TestClockRunStopWatch(GaiaTestCase):
+
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+
+        self.clock = Clock(self.marionette)
+        self.clock.launch()
+
+    def test_click_run_stopwatch_laps(self):
+
+        # Switch to the stopwatch view.
```

Comment:
I think we can get rid of all the comments in this file. One comment is a duplication of the following code line. If one day, the test needed to be changed, we'll have 2 places to maintain for each change. Some people might forget to change the comment => The comment will lie, which will confuse future readers.

To prevent that, the code should be self-explanatory, which is mostly the case here.